### DISCUSSION TITLE
extra line in a cell

### DISCUSSION THREAD

**Discussion by 88arvin** [jsvine/pdfplumber#787] [Q&A]
I am trying to extract a table from the pdf. The data in some cells in the pdf is split into multiple lines, so while extracting the data, the data gets split into separate cells.

After using debug_tablefinder(), I observed that there are some more lines that divide the cell into two distinct cells.

How to remove these extra lines?

sample - [https://filetransfer.io/data-package/j8R9QNfp#link](url)

**Comment by jsvine** [comment]
Fantastic to hear that `.debug_tablefinder()` helped you to figure out the problem. To resolve it, I'd suggest these steps:

1. Examine the values in `page.lines` and `page.rects`. Can you determine which properties are associated with the lines you *don't* want? The answer will differ depending on the PDF, but often you can tell by the `width`, `stroking_color`, or `non_stroking_color` properties.
2. Use `filtered = page.filter(lambda obj: ...)` (see README for explanation) to create a version of the page that does not have those lines/rects. For example, `filtered = page.filter(lambda obj: not (obj["object_type"] == "line" and obj["width"] < 200))`.

**Reply by 88arvin** [reply]
[https://postimg.cc/zL6tW42F](url)
I'm obtaining this after using the page.rects

**Comment by samkit-jain** [comment]
@88arvin You can try filtering out the invisible lines similar to the one explained [here](https://github.com/jsvine/pdfplumber/issues/311#issuecomment-726958591). To help further, would appreciate if you could share reproducible code and the PDF (shared link is asking to create an account).
